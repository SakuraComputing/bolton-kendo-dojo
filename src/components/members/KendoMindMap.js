import React, { Component } from 'react';

class KendoMindMap extends Component {

  render() {
    return (
      <div className="content-container mindmap-layout">
        <img src="/img/background/KendoMindMap.png" alt=""/>
      </div>
    )
  }
}
export default KendoMindMap;
