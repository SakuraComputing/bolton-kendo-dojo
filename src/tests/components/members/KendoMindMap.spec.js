import React from 'react';
import { shallow } from 'enzyme';
import KendoMindMap from '../../../components/members/KendoMindMap';


describe('Mind Map Component', () => {

    let kendoMindMap;

    beforeEach(() => {
        kendoMindMap = shallow(<KendoMindMap />)
    })

    it('should redner correctly', () => {
        expect(kendoMindMap).toMatchSnapshot();
    });
});