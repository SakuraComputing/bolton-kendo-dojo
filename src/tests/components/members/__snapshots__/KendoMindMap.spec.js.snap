// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mind Map Component should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <KendoMindMap />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <img
        alt=""
        src="/img/background/KendoMindMap.png"
      />,
      "className": "content-container mindmap-layout",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "alt": "",
        "src": "/img/background/KendoMindMap.png",
      },
      "ref": null,
      "rendered": null,
      "type": "img",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <img
          alt=""
          src="/img/background/KendoMindMap.png"
        />,
        "className": "content-container mindmap-layout",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "",
          "src": "/img/background/KendoMindMap.png",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
  Symbol(enzyme.__childContext__): null,
}
`;
